examples/forms.amp.html

```diff
@@ -116,6 +117,41 @@
 </head>
 <body>
 
+<amp-analytics>
+    <script type="application/json">
+        {
+            "requests": {
+                "test-ping": "https://my-analytics.com/ping?title=${title}&acct=${account}",
```

Comment:
nit: This URL will show an errror in network tab. Slight preference for a real URL like: https://www.ampproject.org/static/img/logo-blue.svg.